### DISCUSSION TITLE
[ALL] 롤링페이퍼와 메시지를 생성할 때, 작성자 id를 넘겨주는 것이 좋을까?

### DISCUSSION THREAD

**Discussion by kth990303** [woowacourse-teams/2022-nae-pyeon#65] [ALL 컨벤션 이슈]
- 3~4주차부터는 로그인 기능이 추가돼 토큰을 넘겨줄 수 있습니다. 헤더의 토큰값으로 충분히 어떤 멤버가 작성했는지 확인이 가능하기 때문에 작성자 id를 넘겨주지 않아도 괜찮습니다. 
- 하지만 인가 기능이 추가되어 관리자 기능이 생긴다면 작성자 id가 다시 필요해질 수도 있습니다. 

작성자 id를 롤링페이퍼/메시지 생성 response에 넘겨주는 것이 좋을까요?

**Comment by asebn1** [comment]
3~4주차에는 id를 넘겨주는것이 중복되는 것 같아서 이번 스프린트에서는 토큰을 활용하고 관리자 기능이 추가될 때 생각해봐도 좋을 것 같아요
